src/de/jost_net/JVerein/gui/action/BuchungBuchungsartZuordnungAction.java

```diff
@@ -111,7 +104,6 @@ public void handleAction(Object context) throws ApplicationException
             }
           }
         }
-        control.getBuchungsList();
         if (ba == null)
         {
           GUI.getStatusBar().setSuccessText("Buchungsarten gel�scht");
```

Comment:
Ich frage mich echt, warum hier das 'ö' in "gelöscht" nicht richtig dargestellt wird, die Sonderumlaute in `BuchungMenu.java` jedoch schon. Na egal.